commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,22 @@
      }
 
     /**
+     * calculates the page number of the result.
+     *
+     * @return the page number of the result.
+     */
+    default long getPage() {
+
+        if (getOffset() == null) {
```

Comment:
I was actually following the same design pattern used on the class, to make it easy to dete defectuous implementations.
But as you said lets remove the check since the field is not nullable. 👍